src/libmongoc/tests/test-mongoc-cluster.c

```diff
@@ -145,7 +147,6 @@ static void
 _test_cluster_node_disconnect (bool pooled)
 {
    mock_server_t *server;
-   const int32_t socket_timeout_ms = 100;
    mongoc_uri_t *uri;
```

Comment:
I observed one failure of the mock server receiving a query in START_QUERY (but only once). My best guess was that sending the message did not succeed in 100ms. But inspecting this test, I don't think anything here relies on socket_timeout_ms being short (nor does it speed up the test).